app/views/workorders/new.html.haml

```diff
@@ -20,7 +20,7 @@
     .actions
       = f.submit "Send"
 - else
-  = form_for @workorder, url: workorders_path(id: @help_requests.id),method: 'post' do |f|
+  = form_for @workorder, url: workorders_path(help_request_id: @help_requests.id),method: 'post' do |f|
```

Comment:
If this is one object, it should be called `@help_request` - no S. I expect `@help_requests` to be an array of requests, so you wouldn't be able to call `id` on it.